khelo/lib/ui/flow/team/detail/components/team_detail_stat_content.dart

```diff
@@ -35,387 +38,102 @@ class TeamDetailStatContent extends ConsumerWidget {
-              title: context.l10n.team_detail_won_title,
-              count: win.toDouble(),
-              tintColor: context.colorScheme.positive),
-        ),
-        const SizedBox(
-          width: 16,
-        ),
-        Expanded(
-          child: _matchResultCountCell(context,
-              title: context.l10n.team_detail_lost_title,
-              count: lost.toDouble(),
-              tintColor: context.colorScheme.alert),
-        ),
-        const SizedBox(
-          width: 16,
-        ),
-        Expanded(
-          child: _matchResultCountCell(context,
-              title: context.l10n.team_detail_tied_title,
-              count: tie.toDouble(),
-              tintColor: context.colorScheme.warning),
-        ),
-      ],
-    );
-  }
-
-  (int, int, int) _calculateTotalMatchResultCount(TeamDetailState state) {
-    if (state.matches == null) {
-      return (0, 0, 0);
-    }
-
-    var win = 0;
-    var lost = 0;
-    var tie = 0;
-
-    for (final match in state.matches!
-        .where((element) => element.match_status == MatchStatus.finish)) {
-      final firstTeam = match.toss_decision == TossDecision.bat
-          ? match.teams
-              .firstWhere((element) => element.team.id == match.toss_winner_id)
-          : match.teams
-              .firstWhere((element) => element.team.id != match.toss_winner_id);
-      final secondTeam = match.teams
-          .firstWhere((element) => element.team.id != firstTeam.team.id);
-
-      if (firstTeam.run == secondTeam.run) {
-        tie += 1;
-      } else if (firstTeam.run > secondTeam.run) {
-        if (firstTeam.team.id == state.team?.id) {
-          win += 1;
-        } else {
-          lost += 1;
-        }
-      } else {
-        if (secondTeam.team.id == state.team?.id) {
-          win += 1;
-        } else {
-          lost += 1;
-        }
-      }
-    }
-
-    return (win, lost, tie);
-  }
-
-  Widget _matchResultCountCell(
-    BuildContext context, {
-    required String title,
-    required double count,
-    bool showFraction = false,
-    Color? tintColor,
-  }) {
     return Container(
-      padding: const EdgeInsets.all(12),
+      padding: const EdgeInsets.all(16),
       decoration: BoxDecoration(
-        color: tintColor?.withOpacity(0.5) ??
-            context.colorScheme.containerLowOnSurface,
-        borderRadius: BorderRadius.circular(20),
+        color: context.colorScheme.containerLow,
+        borderRadius: BorderRadius.circular(16),
       ),
-      child: Text.rich(
-        TextSpan(
-            text:
-                "${showFraction ? "${count.toStringAsFixed(2)}%" : count.toInt().toString()} ",
-            style: AppTextStyle.header1
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            '$playedMatches ${playedMatches == 1 ? context.l10n.team_detail_match_tab_title : context.l10n.my_game_matches_tab_title}',
```

Comment:
I didn't get it, what does it do? if it's about showing singular - plural form(match - matches) then can we define string in assets and use plural in it.